src/types/Quote.ts

```diff
@@ -0,0 +1,32 @@
+/**
+ * @module TypeInterfaces/quote
+ */
+/**
+ * Dummy comment needed for correct doc display, do not remove.
+ */
+
+import ICType from './CType'
+
+export interface ICostBreakdown {
+  tax: number
+  net: number
+  gross: number
+}
+export default interface IQuote {
+  attesterAddress: string
+  cTypeHash: ICType['hash']
+  cost: ICostBreakdown
+  currency: string
+  quoteTimeframe: Date
+  termsAndConditions: string
+  version: string
```

Comment:
Let's rename it to `specVersion` or similar

Reply:
Done.